dependency_support/load_external.bzl

```diff
@@ -330,3 +321,19 @@ def load_external_repositories():
         urls = ["https://github.com/bazelbuild/rules_pkg/releases/download/1.0.0/rules_pkg-1.0.0.tar.gz"],
         sha256 = "cad05f864a32799f6f9022891de91ac78f30e0fa07dc68abac92a628121b5b11",
     )
+
+    # Used in C++ tests of the ZSTD Module
+    # Transitive dependency of fuzztest (required by riegeli in fuzztest workspace)
+    # Version 1.5.6 released on 30.03.2024
+    # https://github.com/facebook/zstd/releases/tag/v1.5.6
+    # Updated 19.07.2024
```

Comment:
Make that an ISO8601 date format: 2024-07-19

Reply:
Done